<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnailClass</class>
 <widget class="QMainWindow" name="SnailClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snail</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="5,1">
    <item row="0" column="0">
     <widget class="QScaledGraphView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnUploadPoints">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdjGrid">
        <property name="text">
         <string>Grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdjustImg">
        <property name="text">
         <string>Adjust</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QScaledGraphView</class>
   <extends>QGraphicsView</extends>
   <header>qscaledgraphview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="snail.qrc"/>
 </resources>
 <connections/>
</ui>
